testplan/testing/multitest/result.py

```diff
@@ -1608,7 +1608,13 @@ def log_code(self, code, language="python", description=None):
         return entry
 
     @assertion
-    def fail(self, description, category=None, flag=None):
+    def fail(
```

Comment:
We usually use result.log like this:
result.log("This is a log assertion")
result.log(cmd_output, description="execute {cmd}")

I would expect result.fail to exhibits same usage and behavior. Thus shall we change the order of the parameters, and make it consistent with result.log?